server/middleware/auth.ts

```
@@ -4,6 +4,15 @@ import { User } from "../models/user";
 
 const secret = process.env.JWT_SECRET as string;
 
+function getAuthUser(res: Response): User | undefined {
+  if (!res.locals.user) {
+    res.status(401).json({ message: "User is not authorized to perform this request" })
+    return;
```

Maybe this should raise an error? Create a new error class `Unauthorized`